### ISSUE TITLE
Upgrade throws Browser monitor errors

### ISSUE THREAD

**Issue by TheLooostOne** [dynatrace-oss/terraform-provider-dynatrace#574]
**Describe the bug**
Hi, 
I'm currently migrating from 1.44.1 to 1.69.0. No code has been changed regarding the browser monitor resource but I still get the following error :

![image](https://github.com/user-attachments/assets/7879e2a2-aecb-47c5-9d3b-884fdf08fd81)

Here is the initial code :
```
resource "dynatrace_browser_monitor" "lobby" { 
  name      = "AZ"
  frequency = var.requests_frequency
  enabled   = var.monitors_enabled
  locations = ["${var.location}"]
  anomaly_detection {
    loading_time_thresholds {
    }
    outage_handling {
      local_outage = true
      local_outage_policy {
        affected_locations = 1
        consecutive_runs   = var.consecutive_runs
      }
    }
  }
  key_performance_metrics {
    load_action_kpm = "VISUALLY_COMPLETE"
    xhr_action_kpm  = "VISUALLY_COMPLETE"
  }
  script {
    type = "clickpath"
    events {
      event {
        description = "Loading of Lobby"
        navigate {
          url = "https://az.com"
          wait {
            wait_for = "page_complete"
          }
        }
      }
    }
  }
  tags {
    tag {
      context = "CONTEXTLESS"
      key     = "az"
    }
    tag {
      context = "CONTEXTLESS"
      key     = "testing"
    }
  }
}
```

To Reproduce

Create new dynatrace_browser_monitor resource
Apply the changes

Expected behavior
No errors should appear.

Desktop (please complete the following information):

OS: Windows 11 Enterprise
Browser : chrome
Version : 23H2

**Comment by kishikawa12** [comment]
Hi @TheLooostOne,

I've tried replicating the error, but haven't had any luck. I've tried both 1.44.1 and 1.69.0 but succeeded with the executions by hardcoding the four variables in your example. 

Are you able to validate your variables are set properly? I tried intentionally breaking it on my end but I'm unable to run into the same `sizes must be between 1 and 2000` error.

If you continue to run into issues, are you able to enable the following environment variables and send the collected logs to terraform@dynatrace.com?

```
export DYNATRACE_DEBUG=true
export DYNATRACE_LOG_HTTP=terraform-provider-dynatrace.http.log
export DYNATRACE_HTTP_RESPONSE=true
```

Thanks!
